ksqldb-functional-tests/src/test/resources/query-validation-tests/source-stream.json

```diff
@@ -66,6 +66,28 @@
         "type": "io.confluent.ksql.util.KsqlException",
         "message": "Cannot delete topic for read-only source: INPUT"
       }
+    },
+    {
+      "name": "rejects CREATE OR REPLACE SOURCE ... statements for source streams",
```

Comment:
Should we add more cases (same for table)
1. `INPUT` does not exist
2. `INPUT` is create as regular STREAM and `CREATE AND REPLACE SOURCE input` fails